src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/JpaReviewSchedulePersistenceAdapterTest.kt

```diff
@@ -182,37 +184,37 @@ class JpaReviewSchedulePersistenceAdapterTest {
       verify(reviewScheduleEntityMapper).fromEntityToDomain(reviewScheduleEntity)
       verify(reviewScheduleHistoryRepository).save(any())
     }
-  }
 
-  @Test
-  fun `should not create review schedule given prisoner already has an active Review Schedule`() {
-    // Given
-    val prisonNumber = randomValidPrisonNumber()
+    @Test
```

Comment:
Minor change to this test, in that it should have been nested in the inner class, but it wasnt.
I've moved it up into the parent inner class, so the diff here is really just an indentation change 👍